frontend/components/pod_components/HomeScreen.js

```diff
@@ -6,83 +6,91 @@ import { getAccessToken } from '../../utils/auth.js';
+                "headers": {
                     "Authorization": "Bearer " + accessToken
                 }
-            })
-        )
+        }))
         .then(async response => {
             let data = await response.json();
+            console.log(data);
             this.setState({
-                Trainee_progress: data.records[0].TR_Competency_Completed__c + data.records[0].TR_CareerExpl_Completed__c + data.records[0].TR_LifeEssentials_Completed__c,
-                Associate_progress: 0,
-                Partner_progress: 0
+                Trainee_progress: data.Trainee_complete,
+                Trainee_total: data.Trainee_total,
+                Associate_progress: data.Associate_complete,
+                Associate_total: data.Associate_total,
+                Partner_progress: 1, // should change to *data.Partner_complete*
+                Partner_total: data.Partner_total,
             })
         })
-        .catch(function (error) {
-            console.log(error);
+        .catch(function (error){
+            print("There is no pod data to display. Please contact your manager or More Than Words administrator if you think this is an error.")
         });
     }
 
     render(){
-        return (
-        <ScrollView style={styles.scrollView}>
-            <SafeAreaView style={styles.container}>
-            <TouchableOpacity 
-                style={styles.block} 
-                onPress={() => this.props.navigation.navigate('Trainee Pod', {
-                    pod: 'Trainee'
-                })}
-            >
-                <Text style={styles.blockText}>
-                    Trainee 
-                </Text>
-                <ProgressBar progress={this.state.Trainee_progress} total_outcomes={TRAINEE_TOTAL_OUTCOMES} />
-            </TouchableOpacity>
+        let T_displayBlock = styles.Block;
+        let A_displayBlock = styles.Block;
+        let P_displayBlock = styles.Block;
+        let T_displayBlockText = styles.BlockText;
+        let A_displayBlockText = styles.BlockText;
+        let P_displayBlockText = styles.BlockText;
+        if (this.state.Trainee_progress < this.state.Trainee_total){ // highlight Trainee
+            T_displayBlock = styles.highlightBlock;
+            T_displayBlockText = styles.highlightBlockText;
+        } else if(this.state.Associate_progress < this.state.Associate_total){ // highlight Associate
+            A_displayBlock = styles.highlightBlock;
+            A_displayBlockText = styles.highlightBlockText;
+        } else { // highlight Partner
+            P_displayBlock = styles.highlightBlock;
+            P_displayBlockText = styles.highlightBlockText;
+        }
+           
+        return(
+        <SafeAreaView style={styles.container}>
+        <TouchableOpacity 
+            style={T_displayBlock} 
+            onPress={() => this.props.navigation.navigate('Trainee Pod', {
+                pod: 'Trainee'
+            })}
```

Comment:
This is the code that I added in order for the focus area page to be able to navigate to the outcomes screen because you need to pass the current pod as a prop